manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+  "name": "Cuttle.cards",
+  "short_name": "Cuttle",
+  "description": "The deepest card game under the sea!",
+  "theme_color": "#FFF4D7",
+  "background_color": "#FFF4D7",
+  "display": "fullscreen",
+  "id": "/",
+  "start_url": "/",
```

Comment:
Do these need to be any different for production?

Reply:
I don't _think_ @seriouslysean any thoughts?

Reply:
Nah this seems fine to me, we don't run in any sort of subdirectory situation.